scripts/kubectl-kubestellar-prep_for_syncer

```diff
@@ -114,26 +117,26 @@ elif ! [[ "$espw" =~ [a-z0-9].* ]]; then
     exit 1
 fi
 
-trap 'kubectl ws "${kubectl_flags[@]}" "$original_ws"' EXIT
 
-kubectl ws "${kubectl_flags[@]}" "$imw"
-
-echo "--- current directory is $PWD"
-cwsi=$(kubectl ws "${kubectl_flags[@]}" .)
-cwsi=${cwsi#*'"'}; cwsi=${cwsi%'"'*}
-echo "current ws is $cwsi"
-echo "imw is $imw"
+if [ "$silent" == "true" ]; then
+    trap 'kubectl ws "${kubectl_flags[@]}" "$original_ws" > /dev/null' EXIT
+else
+    trap 'kubectl ws "${kubectl_flags[@]}" "$original_ws"' EXIT
+fi
 
-echo "current cwsi is $cwsi"
-if ! kubectl get apibinding "edge.kcp.io" &> /dev/null; then
-   kubectl kcp bind apiexport root:espw:edge.kcp.io
-   echo "bound apiexport root:espw:edge.kcp.io to $cwsi:edge.kcp.io"
-else echo "edge.kcp.io apibinding exists in workspace $cwsi"
-fi	 
+if [ "$silent" == "true" ]; then
+    kubectl ws "${kubectl_flags[@]}" "$imw" > /dev/null
+else
+    kubectl ws "${kubectl_flags[@]}" "$imw"
+fi
 
-mbwsname=$(kubectl "${kubectl_flags[@]}" get synctargets.edge.kcp.io "$stname" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}-mb-{.metadata.uid}")
+mbwsname=$(kubectl "${kubectl_flags[@]}" get SyncTarget "$stname" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}-mb-{.metadata.uid}")
```

Comment:
This was a step backwards, undoing @fileppb's work in #814 on copying SyncTarget and Location to the KubeStellar repo.